auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -329,6 +348,22 @@ public boolean checkRight(String actorCrn, String userCrn, String right, Optiona
         return checkRight(actorCrn, userCrn, right, null, requestId);
     }
 
+    /**
+     * Retrieves whether the member has the specified rights.
+     *
+     * @param actorCrn the CRN of the actor
+     * @param memberCrn the CRN of the member
+     * @param rightChecks the rights to check
+     * @param requestId an optional request id
+     * @return a list of booleans indicating whether the member has the specified rights
+     */
+    public List<Boolean> hasRights(String actorCrn, String memberCrn, List<AuthorizationProto.RightCheck> rightChecks, Optional<String> requestId) {
+        try (ManagedChannelWrapper channelWrapper = makeWrapper()) {
+            AuthorizationClient client = new AuthorizationClient(channelWrapper.getChannel(), actorCrn);
+            return client.hasRights(requestId.orElse(UUID.randomUUID().toString()), memberCrn, rightChecks);
```

Comment:
we are currently not making call to UMS in case of internal CRN because right now everything allowed witn internalCRN afaik, there is an `if (InternalCrnBuilder.isInternalCrn(actorCrn))` in checkRight call for it, I think we should use that here also

Reply:
Also, many of the UMS calls fail with internal CRN.

Reply:
Not to mention that we are overloading network with internalCrn requests to UMS, while we know that everything is allowed with internalCrn

Reply:
I just looked at the checkRights code and it's wrong. You should check if the `InternalCrnBuilder.isInternalCrn(userCrn)` instead of actorCrn. I think that the only caller right now calls checkRight with actorCrn == userCrn, so we've been lucky so far.

@keyki Can you give an example of a UMS call that fails with the internal CRN? Do you mean internal CRN as the actor (grpc context) or as the member (part of request)? I'd like to understand the issue because it doesn't make sense to me. Could it be that you are creating internal CRNs with a service other than IAM?

Reply:
> Not to mention that we are overloading network with internalCrn requests to UMS, while we know that everything is allowed with internalCrn

I'm setting the actorCrn (grpc context) to internal Crn, but requesting whether the memberCrn has rights. I should never repeat a request within a user sync operation.